modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -177,7 +177,9 @@ export default class ColorInput extends React.Component<ColorInputProps> {
             icon={checkSmallIcon}
           />
         ) : null}
-        <PoundSignPrefix disabled={disabled}>#</PoundSignPrefix>
+        <PoundSignPrefix aria-hidden={true} disabled={disabled}>
```

Comment:
Not sure if this is a part of this PR, but I had to look up what `PoundSignPrefix` was... (part of the problem with the Styled component API). Turns out this is a `span` element which doesn't have a `disabled` attribute. What is the `disabled={true}` supposed to do?

Reply:
Oh, I see. It is for styling only: https://github.com/Workday/canvas-kit/pull/523/files#diff-f2800e7c1a68cd10843f1a74fb5e1e48R94-R97

Reply:
it changes the color of the pound sign... which in no way is that clear.